platform/ios/CHANGELOG.md

```diff
@@ -2,10 +2,78 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
-## 3.7.7
+## 3.7.7 - May 3, 2018
 
 * Fixed a crash when removing an `MGLOfflinePack`. ([#6092](https://github.com/mapbox/mapbox-gl-native/issues/6092))
 
+## 4.0.0 - April 19, 2018
```

Comment:
The blurb above shouldn't point to the PR instead? https://github.com/mapbox/mapbox-gl-native/pull/11821